tests/tests_integration/test_api/test_model_hosting/test_source_packages.py

```diff
@@ -12,6 +12,7 @@
 SOURCE_PACKAGES_API = CogniteClient().model_hosting.source_packages
 
 
+@pytest.mark.skip(reason="wait for model hosting v1")
```

Comment:
Shouldn't/couldn't this run in playground and be ok?

Reply:
All methods now expect that the "data"-wrapper is removed from the json-bodies and will therefore fail until we deploy v1.

Reply:
All the integration tests run against Greenfield. So this should pass.